dart/dynamics/FreeJoint.cpp

```diff
@@ -58,14 +58,32 @@ FreeJoint::~FreeJoint()
 }
 
 //==============================================================================
+Eigen::Vector6d FreeJoint::convertToPositions(const Eigen::Isometry3d& _tf)
+{
+  Eigen::Vector6d x;
+  x.head<3>() = math::logMap(_tf.rotation());
```

Comment:
I've been using `Eigen::Isometry3d::linear()` to get the rotation part of a transformation. Do you know what's the difference between `linear()` and `rotation()`?